### PR TITLE
#14 manage bookings UI

### LINKED ISSUE #14: FE: Manage Bookings UI
Follow mockup
-Calendar view
\- active upcoming bookings

### PR DESCRIPTION
**Pull request by joelmackenz**
### What this PR does (required):
- Adds the ManageBookings UI and a Bookings element.
- A rendered Booking can accept "outlined", "accepted", or "large" props to change its appearance.

### Screenshots / Videos (required):

https://user-images.githubusercontent.com/66138947/122471548-41154100-cf74-11eb-8d10-d9ed3a7661f3.mov

### Any information needed to test this feature (required):
- Create access to the ManageBookings.tsx page (fast & easy way: change <Route /> in App.tsx to <ManageBookings />)

### Any issues with the current functionality (optional):
-N/A

### CODE REVIEW

**Review by g4rry420 — COMMENTED** [review]
Everything looks good Joel and it is responsive also. Just a few suggestions.
- Please, make sure font size is small on mobile viewport.
- Also, imo, it will be good UX experience if we put past booking in a separate column just like Next Booking.

**Comment by joelmackenz** [comment]
@g4rry420 Thanks Gurkiran! I'll make that font change. I agree re: the past bookings in a separate column -- that's different from the mockup, but if we're all on board, I can make that change.

File: client/src/pages/ManageBookings/useStyles.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  bookingAvatar: {
+    marginRight: '1rem',
+    marginTop: '1rem',
+    marginBottom: '1rem',
+  },
+  bookingProfileGrid: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
```
Comment (moffatethan) [inline client/src/pages/ManageBookings/useStyles.ts:37]: I don't see this being used in ManageBookings, if there are any unused styles here they should be removed.

**Review by moffatethan — APPROVED** [review]
This looks good, I gave you a comment on the style file. Good job on this!

File: client/src/pages/ManageBookings/useStyles.ts (hunk 1 above)
Comment (joelmackenz) [reply client/src/pages/ManageBookings/useStyles.ts:37]: This is actually being used in Booking.tsx, on line 28.

**Comment by joelmackenz** [comment]
@moffatethan Thank you!

**Comment by joelmackenz** [comment]
@g4rry420 Hey! In regards to your comment on smaller text, do you mean that we should be adding something beyond responsiveFontSizes in our theme? That is already handling the size of the text on mobile and the other screen sizes. Or am I missing something?

I made your other suggested change, putting Past Interviews into its own card, in the integration PR.

**Comment by joelmackenz** [comment]
I'm going to merge this for now, and I can make any other necessary changes to the text size on the integration PR!

**Pull request merged**